EveripediaNetwork/ep-ui
# Added hover state to select menu on homepage

# Added hover state to select menu on homepage

Added a hover state to the select menu on the homepage to improve user experience.

## How should this be tested?

1. Navigate to the homepage.
2. Hover over the select menu in the Trending Wikis section.
3. Verify that the hover state is applied correctly.

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3033
